code/game/objects/items/weapons/shards.dm

```diff
@@ -87,6 +87,7 @@
 				H.Weaken(3)
 				if(affecting.take_damage(5, 0))
 					H.UpdateDamageIcon()
+					affecting.embed(src)
```

Comment:
Glass getting stuck in your foot? I'd say 50-80%. People make broken glass sound so dangerous that you shouldn't even touch it with bare hands so stepping on it should be bad